c10/util/Unicode.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#if defined(_WIN32)
+#include <string>
+#include <c10/util/win32-headers.h>
+#include <c10/util/Exception.h>
+#endif
+
+namespace c10 {
+#if defined(_WIN32)
+    inline std::wstring u8u16(const std::string& str) {
```

Comment:
Why indentation here? (Rest of the codebase does not indent code inside namespace)

Reply:
Yeah it's a mistake. A result of copy -paste.